src/scripts/OSFramework/OSUI/Pattern/Dropdown/ServerSide/DropdownServerSide.ts

```diff
@@ -1017,7 +1013,6 @@ namespace OSFramework.OSUI.Patterns.Dropdown.ServerSide {
 			this.setCallbacks();
 			this.setHtmlElements();
 			this._setInitialOptions();
-			super.finishBuild();
```

Comment:
This has been removed from here, and placed into the right spot in order to grant the finishBuild since the trigger of the Initialized event has been replaced into this method.